docs/helm-broker/03-03-create-addons-repo.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Create addons repository
+type: Details
+---
+
+The repository in which you create your own addons must be an HTTPS server with a specific structure so that the Helm Broker can fetch addons from it. Your remote addons repository can contain many addons, each one compressed to the `.tgz` format and defined in `index.yaml` files. Depending on your needs and preferences, you can create one or more `index.yaml` files to categorize your addons. The repository structure looks as follows:
```

Comment:
The repository must be a HTTPS server? I may be wrong, but shouldn't the repository be stored on a HTTPS server? I'm not sure if you can say that a repository IS a server.

Reply:
The sentence below the repo structure makes me think I'm right.

Reply:
Yup, you are right, thanks for spotting that.